GH action for linting is broken

**Describe the bug**
The GH action that runs the linting task is broken. It fails during the "install prereqs" step while attempting to install the markdown linter (mdl).

```
ERROR:  Error installing mdl:
	The last version of chef-utils (>= 0) to support your Ruby & RubyGems was 16.6.14. Try installing it with `gem install chef-utils -v 16.6.14` and then running the current command again
	chef-utils requires Ruby version >= 2.6.0. The current ruby version is 2.5.1.57.
Successfully installed asciidoctor-2.0.12
Parsing documentation for asciidoctor-2.0.12
Installing ri documentation for asciidoctor-2.0.12
Done installing documentation for asciidoctor after 2 seconds
Successfully installed rexml-3.2.4
Successfully installed kramdown-2.3.0
Successfully installed kramdown-parser-gfm-1.1.0
Successfully installed mixlib-cli-2.1.8
Successfully installed tomlrb-2.0.0
Successfully installed mixlib-config-3.0.9
1 gem installed
Error: Process completed with exit code 1.
```

**Steps to reproduce**
Trigger a CI run in the repo

**Expected behavior**
`mdl` should properly install, and the tests should run to completion

**Actual results**
mdl fails to install, causing the lint step to fail

**Additional context**
It appears some package versions changed underneath us as it doesn't seem to have been caused by any commits. (There's the same problem over in the SnapScheduler repo)
Perhaps moving to Ubuntu-20.04 (a little early?) would fix the problem. https://github.com/actions/virtual-environments/issues/1816

--- a/.github/workflows/documentation.yml
+++ b/.github/workflows/documentation.yml
@@ -13,7 +13,7 @@ on:  # yamllint disable-line rule:truthy
 jobs:
   build:
     name: Build
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
 
     steps:
       - name: Checkout source
@@ -23,7 +23,7 @@ jobs:
         run: |
           echo 'APT::Acquire::Retries "5";' | sudo tee /etc/apt/apt.conf.d/80-retries
           sudo apt-get update
-          sudo DEBIAN_FRONTEND=noninteractive apt-get install -y python-pip python3-venv
+          sudo DEBIAN_FRONTEND=noninteractive apt-get install -y python3-pip python3-venv
 
       - name: Build docs
         run: |

--- a/.github/workflows/mover-rsync.yml
+++ b/.github/workflows/mover-rsync.yml
@@ -16,7 +16,7 @@ env:
 jobs:
   lint:
     name: Lint-mover-rsync
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
 
     steps:
       - name: Checkout source
@@ -26,15 +26,15 @@ jobs:
         run: |
           echo 'APT::Acquire::Retries "5";' | sudo tee /etc/apt/apt.conf.d/80-retries
           sudo apt-get update
-          sudo DEBIAN_FRONTEND=noninteractive apt-get install -y python-pip ruby
+          sudo DEBIAN_FRONTEND=noninteractive apt-get install -y python3-pip ruby
           sudo gem install asciidoctor mdl
-          sudo pip install yamllint
+          sudo pip3 install yamllint
       - name: Run linters
         run: ./.ci-scripts/pre-commit.sh --require-all
 
   build:
     name: Build-mover-rsync
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
 
     steps:
       - name: Checkout source
@@ -58,7 +58,7 @@ jobs:
     if: >
       (github.event_name == 'push' || github.event_name == 'schedule') &&
       (github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags/v'))
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
 
     steps:
       - name: Load container artifact

--- a/.github/workflows/operator.yml
+++ b/.github/workflows/operator.yml
@@ -16,7 +16,7 @@ env:
 jobs:
   lint:
     name: Lint
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
 
     steps:
       - name: Checkout source
@@ -26,15 +26,15 @@ jobs:
         run: |
           echo 'APT::Acquire::Retries "5";' | sudo tee /etc/apt/apt.conf.d/80-retries
           sudo apt-get update
-          sudo DEBIAN_FRONTEND=noninteractive apt-get install -y python-pip ruby
+          sudo DEBIAN_FRONTEND=noninteractive apt-get install -y python3-pip ruby
           sudo gem install asciidoctor mdl
-          sudo pip install yamllint
+          sudo pip3 install yamllint
       - name: Run linters
         run: ./.ci-scripts/pre-commit.sh --require-all
 
   build:
     name: Build-operator
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
 
     steps:
       - name: Checkout source
@@ -65,7 +65,7 @@ jobs:
   e2e:
     name: End-to-end
     needs: build
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     strategy:
       fail-fast: false
       matrix:
@@ -86,7 +86,7 @@ jobs:
     if: >
       (github.event_name == 'push' || github.event_name == 'schedule') &&
       (github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags/v'))
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
 
     steps:
       - name: Load container artifact

